actions/hls.mk

```diff
@@ -74,16 +74,16 @@ $(SOLUTION_NAME): $(objs)
 # Check for register duplication (0x184/Action_Output_o).
 #
 check: $(syn_dir)
-	@echo -n "Checking for critical warnings during HLS synthesis ... "
+	@echo -n "   Checking for critical warnings during HLS synthesis .... "
 	@grep -A8 critical $(SOLUTION_DIR)*/$(SOLUTION_NAME)/$(SOLUTION_NAME).log ; \
 		test $$? = 1
 	@echo "OK"
-	@echo -n "Checking for reserved MMIO area during HLS synthesis ... "
+	@echo -n "   Checking for reserved MMIO area during HLS synthesis ... "
 	@grep -A8 0x17c $(syn_dir)/vhdl/$(WRAPPER)_ctrl_reg_s_axi.vhd | grep reserved > \
 		/dev/null; test $$? = 0
 	@echo "OK"
 
 clean:
-	@$(RM) -r $(SOLUTION_DIR)* run_hls_script.tcl *~ *.log \
+	@$(RM) -r $(SOLUTION_DIR)* run_hls_script.tcl *~ *.log hls_syn_vhdl \
```

Comment:
hls_syn_vhdl is already getting removed in the next line

Reply:
`@for link in $(symlinks); do $(RM) hls_syn_$(link); done` doesn't work!
Only correction I found is:
`@for link in $(symlinks); do $(RM) hls_syn_$(symlinks); done`
Would you accept that ?

Reply:
Ah, I didn't realize that there was a problem with that. Thanks for making me aware of that.
I pushed a fix.